wiki/Tournaments/GBC/GBC_2024_Spring/en.md

```diff
@@ -0,0 +1,221 @@
+6. Every week, the referee creates a multiplayer room at the fixed times, and the players who need to play a match could enter the room.
+7. There is **no warm-up phase** for the match.
+8. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins is allowed.
+   - Custom skin elements **must not** be used to alter core gameplay mechanics.
+
+### Registration
+
+1. Please **fill [this form](https://wj.qq.com/s2/14213350/thxh/)** to register.
+   - Player **must** join [this QQ group](https://jq.qq.com/?_wv=1027&k=zqBeTqMW) or [this Discord server](https://discord.gg/qpJjGuYyC5) after register.
+   - If you cannot use QQ, please use osu! or Discord to contact the organiser and use Discord to communicate.
+2. To ensure valid and serious registrations, every registered user will be manually checked by the host.
+   - **There are no rank limitations for this tournament.**
+   - To be successfully registered, a player must not have violated any [osu! community rules](/wiki/Rules) within the last 12 months.
+   - When registering, the organiser will refer to the player's play counts, heavy derankers and smurfs may be forced to upgrade.
+3. A list of all successfully registered players will be published after the registration phase.
+4. All successfully registered players will be grouped as follows:
+
+| Group | Player's 4K pp |
+| :-: | :-- |
+| EX | 9500 and above |
+| S | 7000 – 8999 |
+| A | 4500 – 6999 |
+| B | 0 – 4499 |
+
+5. If the 4K pp of a player is much higher than the upper limitation of his level, he will be forced to promote into next level. The buffer is *100 pp*.
+6. [Tournament staff members](/wiki/Tournaments/Official_support#staff) must not participate as players in the tournament.
+7. Organisers may force to promote some players that are considered too strong in their levels.
+8. Organisers reserve the right to decline restricted players from participating in the tournament.
+9. It is allowed for players to request to promote their level. They need to send some result screenshotsof next level to the organiser for getting premission.
+
+### Win conditions
+
+1. In the Regular stage, players need to get higher placement in higher level for higher points. The points that can be earned are determined by the placement in every round. The placement according to the average rank for all individual beatmaps in a round.
+2. We have a base points that depends on the placement per round, and a multiplier that depends on the level of players. Here is the calculation method of points.
+
+| Placement | Base points |
+| :-: | --: |
+| 1st place | 20 |
+| 2nd place | 19 |
+| 3rd place | 18 |
+| 4th place | 17 |
+| Top 15% | 16 |
+| Top 22% | 15 |
+| Top 30% | 14 |
+| Top 40% | 13 |
+| Top 50% | 12 |
+| Top 60% | 10 |
+| Top 70% | 8 |
+| Top 80% | 6 |
+| Top 90% | 4 |
+| Top 100% | 2 |
+
+| Level | Multiplier (x is base points) |
```

Comment:
ok it was "group" before, is "level" the same thing? do players move between levels/groups during the tournament?

Reply:
Yeah level and group are same thing and always be used together, the prize will calculation separated for every round if they upgrade or reduce. For example, if level B prize is 20 and level A prize is 30, and you played 2 rounds level B and 2 rounds level A, you'll get 25.

Reply:
has changed all the level to group